opennebula/resource_opennebula_template.go

```diff
@@ -48,6 +48,36 @@ func resourceOpennebulaTemplate() *schema.Resource {
 			"os":       osSchema(),
 			"vmgroup":  vmGroupSchema(),
 			"tags":     tagsSchema(),
+			"raw": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "Low-level hypervisor tuning",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
why optional ? It is mandatory no ?

Reply:
I probably misunderstood the doc at first (https://docs.opennebula.io/5.12/operation/references/template.html#hypervisor-section), in sunstone it seems required